### PR TITLE
Script working but not sure how to test it

### PR DESCRIPTION
**Pull request by beaugaines**
Any tips?

### CODE REVIEW

**Comment by mrhead** [comment]
Just check other examples. And focus on testing one public method each in each test...

**Comment by r00k** [comment]
This answer will sound a little snarky, but I think the best way to test things it to write them using TDD in the first place.

Adding tests to code without tests is pretty tricky, and not a very good way to learn testing, in my opinion.

You'll find it's often hard to add tests to untested code. That's because code written without tests isn't written in a way that's easy to test. Makes sense, right?

File: lib/scottish_play.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  end
+end
+
+class ScottishPlay
+
+  def fetch_doc doc
+    Nokogiri::XML.parse(open(doc))
+  end
+
+  def tally_personas doc
```
Comment (qpowell) [inline lib/scottish_play.rb]: Using parentheses on methods that accept arguments could help improve readability of some of your methods here.

More info can be found on the style guide here: https://github.com/thoughtbot/guides/tree/master/style#ruby

Comment (mrhead) [inline lib/scottish_play.rb]: Hello @qpowell,

I totally agree. Nowadays I use parentheses for such a cases.

**Pull request closed without merging**